CHANGELOG.md

```diff
@@ -1,5 +1,52 @@
 # Change Log
 
+## [1.2.0b1] - 2022-03-16
```

Comment:
@finswimmer a few changes missing, otherwise looking good.

- https://github.com/python-poetry/poetry/commit/ba5649e51783dc819680dc8a6f9ad78cb12245b2
- https://github.com/python-poetry/poetry/commit/0761aa7f2b7b0cad53c66f4bae0ce5e276994c06
- https://github.com/python-poetry/poetry/pull/3811

Reply:
Added :+1: